_tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var StudentsAndGraduates = require( '../../page_objects/page_careers-students-and-graduates.js' );
+
+describe( 'Beta: Careers/Student-and-graduates', function() {
```

Comment:
Is describing this with `Beta: ` a convention? Minor, but technically this code is not on beta yet so is not describing a beta test.